apps/challenges/views.py

```diff
@@ -245,6 +246,30 @@ def challenge_detail(request, challenge_host_team_pk, challenge_pk):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+@api_view(['GET'])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def team_name_for_challenge(request, challenge_pk):
```

Comment:
Can we please change the name to `def participant_team_detail_for_challenge` because it gives the complete details of the team and not just the name.